packages/jobs/lib/execution/webhook.ts

```diff
@@ -188,7 +188,7 @@ export async function handleWebhookSuccess({ nangoProps }: { nangoProps: NangoPr
     const logCtx = await logContextGetter.get({ id: String(nangoProps.activityLogId) });
     const environment = await environmentService.getById(nangoProps.environmentId);
     if (environment) {
-        for (const model of nangoProps.syncConfig.models) {
+        for (const model of nangoProps.syncConfig.models!) {
```

Comment:
Is there any way for us to remove this non-null assertion?

Reply:
yes you are right, this has been noted here 👍🏻 
https://github.com/NangoHQ/nango/blob/c62d19625cc409fcfb7618a0b9811cdfddd6eb66/packages/types/lib/syncConfigs/db.ts#L12